src/components/DatePicker.js

```diff
@@ -234,72 +184,43 @@ const DatePicker = React.createClass({
 
   styles () {
     return {
-      calendarDay: {
-        color: StyleConstants.Colors.FOG,
-        float: 'left',
-        height: 30,
-        marginBottom: 2,
+      component: Object.assign({
+        boxSizing: 'border-box',
+        backgroundColor: StyleConstants.Colors.WHITE,
+        color: StyleConstants.Colors.BLACK,
+        display: 'inline-block',
+        fontFamily: StyleConstants.FontFamily,
+        fontSize: StyleConstants.FontSizes.MEDIUM,
         position: 'relative',
-        width: 35
-      },
-      calendarDayContent: {
+        width: '100%',
+        borderColor: this.state.showCalendar ? this.props.primaryColor : StyleConstants.Colors.FOG,
         borderRadius: 3,
-        height: 30,
-        left: '50%',
-        position: 'absolute',
-        top: '50%',
-        transform: 'translateY(-50%) translateX(-50%)',
-        width: 35,
+        borderStyle: 'solid',
+        borderWidth: 1
+      }, this.props.style),
 
-        ':hover': {
-          border: '1px solid' + this.props.primaryColor,
-          borderRadius: 3,
-          cursor: 'pointer'
-        }
-      },
-      calendarDayDisabled: {
-        ':hover': {
-          background: 'none',
-          color: StyleConstants.Colors.FOG
-        }
-      },
-      calendarDayText: {
-        left: '50%',
-        position: 'absolute',
-        top: '50%',
-        transform: 'translateY(-50%) translateX(-50%)'
-      },
-      calendarHeader: {
-        clear: 'both',
-        color: StyleConstants.Colors.CHARCOAL,
-        fontSize: StyleConstants.FontSizes.LARGE,
-        height: 30,
-        marginBottom: 15,
+      // Selected Date styles
+      selectedDateWrapper: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        cursor: 'pointer',
         position: 'relative',
-        textAlign: 'center'
+        padding: 7
```

Comment:
The padding here should be match the dropdown select and other inputs. Product pointed out to me in the case where a form has lots of inputs we want them to match.